src/components/Maps/Marker.tsx

```diff
@@ -22,25 +34,36 @@ const Marker = ({
   lng,
   /* eslint-enable */
   isCurrentLocation,
-  otherClicked,
+  type,
+  clicked,
```

Comment:
nit: boolean props should also be prefaced with `is`
```suggestion
  isClicked,
```